storage/pebble/open.go

```diff
@@ -106,20 +106,6 @@ func IsPebbleInitialized(folderPath string) error {
 		}
 	}
 
-	// Optionally, validate the CURRENT file references a MANIFEST file
-	currentPath := filepath.Join(folderPath, "CURRENT")
```

Comment:
Pebble checkpoint file doesn't have `CURRENT`. 

The `MANIFEST-*` is enough to confirm the folder has pebble db initialized.